subiquity/server/controllers/filesystem.py

```diff
@@ -410,14 +420,37 @@ def disallowed_encryption(msg):
 
         return info
 
+    def _maybe_disable_encryption(self, info: VariationInfo) -> None:
+        if self.model.bootloader != Bootloader.UEFI:
+            log.debug("Disabling core boot based install options on non-UEFI " "system")
```

Comment:
```suggestion
            log.debug("Disabling core boot based install options on non-UEFI system")
```

Reply:
Huh I thought our black config did that!